src/mito2/src/sst/parquet/reader.rs

```diff
@@ -225,7 +229,7 @@ impl ParquetReaderBuilder {
 
         metrics.build_cost = start.elapsed();
 
-        let filters = if let Some(predicate) = &self.predicate {
+        let mut filters = if let Some(predicate) = &self.predicate {
```

Comment:
Is it possible to reserve the capacity for `filters`?

Reply:
> Is it possible to reserve the capacity for `filters`?

Cannot determine the num of time range filters here.